sw/device/lib/dif/dif_kmac_unittest.cc

```diff
@@ -8,6 +8,12 @@
 
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/base/testing/mock_mmio.h"
+#include "sw/device/lib/testing/entropy_testutils.h"
+#include "sw/vendor/llvm_clang_rt_profile/compiler-rt/lib/profile/InstrProfiling.h"
```

Comment:
I think you should probably avoid using this include, please see comments bellow for alignment suggestions.